### PR TITLE
📝 docs: add deploy to Zeabur guide in README

### PR DESCRIPTION
**Pull request by MichaelYuhe**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 🔨 chore
- [x] 📝 docs

#### 🔀 变更说明 | Description of Change

Add a deploy button for deploying Lobe Chat to Zeabur in `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -379,6 +380,22 @@ $ docker run -d -p 3210:3210 \
 
 <br/>
 
+### `C` Deploying with Zeabur
+
+If you want to deploy this service yourself on Vercel, you can follow these steps:
```
Comment (arvinxx) [inline README.md]: Typo

**Review by arvinxx — CHANGES_REQUESTED** [review]
Please also add it to the Readme.zh-CN

File: README.md (hunk 1 above)
Comment (MichaelYuhe) [reply README.md]: Resolved

**Comment by MichaelYuhe** [comment]
Thanks for the review, I've also added it to the Readme.zh-CN @arvinxx ,

**Review by arvinxx — APPROVED** [review]
Great work! Thanks~

**Comment by canisminor1990** [comment]
@MichaelYuhe plz check the deploy link, Error 500

image

**Pull request merged**